react-redux/.eslintrc.json

```diff
@@ -17,6 +17,6 @@
     "react/jsx-filename-extension": ["warn", { "extensions": [".js", ".jsx"] }],
     "import/no-unresolved": "off",
     "no-shadow": "off",
-    "arrow-parens":"off"
+    "arrow-parens": ["error", "as-needed"]
```

Comment:
@jofer86 Could you check this out? 
I will like to know what you think about this...

Reply:
On it, I have a 1:1 session now right after this I will run the tests and see how does stickler behave with that, to avoid mismatches in the local environment and on stickler checks.

Reply:
Also, there's a similar discussion here too https://github.com/microverseinc/linters-config/pull/48#issuecomment-576858776

@efrapp can you check this PR too? :thinking: Seems to be solving the same problem here.

Reply:
Hi @bolah2009, I already tested this rule and it works better than the one I suggested.
For this code:
```
strForEach(c => {
      if (/\W/.test(c)) {
        cipherText += c;
      } else {
        cipherText += alphabet[cipher(alphabet.indexOf(c))];
      }
    });
```
The `"arrow-parens": ["error", "as-needed"]` detected an error when the arrow function passed to the `strForEach` has a single parameter but the config I suggested didn't.

Reply:
Cool, thanks @efrapp

Reply:
It is with pleasure @bolah2009.